core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -273,6 +299,75 @@ export class LoggedOutUser extends BaseUser {
       .toBe(expectedDestinationPageUrl);
   }
 
+  /**
+   * Clicks a link based on its visible text and verifies that the resulting navigation
+   * in the same tab leads to the expected URL.
+   */
+  private async clickLinkAnchorInSameTab(
```

Comment:
Is this function necessary? clickButtonToNavigateToNewPage should do the same thing as passing the text of the button to clickOn() should also works.

Reply:
Oh, my bad! True this function is not needed. 

thanks.